packages/apps/nats/templates/nats.yaml

```diff
@@ -18,6 +40,18 @@ spec:
     nats:
       fullnameOverride: {{ .Release.Name }}
       config:
+        {{- if .Values.users }}
+        {{- if gt (len .Values.users) 0 }}
```

Comment:
Did you meant to check `$passwords`? Because these two checks will have the same effect